pkg/da/data_availability_header.go

```diff
@@ -68,7 +68,8 @@ func ExtendShares(squareSize uint64, shares [][]byte) (*rsmt2d.ExtendedDataSquar
 		)
 	}
 	// here we construct a tree
-	return rsmt2d.ComputeExtendedDataSquare(shares, appconsts.DefaultCodec(), wrapper.NewConstructor(squareSize))
+	// Note: uses the nmt wrapper to construct the tree.
+	return rsmt2d.ComputeExtendedDataSquare(shares, appconsts.DefaultCodec(), nmtwrapper.NewConstructor(squareSize))
```

Comment:
[nit non-blocking]
perhaps we could merge this we the above outdated comment (that doesn't really make sense by itself)?

`// we construct the data root using the nmt wrapper`

Reply:
`// Note: uses the nmt wrapper to construct the tree.` => `// we construct the data root using the nmt wrapper`?

Reply:
yep, makes sense!